tests/integration/conftest.py

```diff
@@ -79,8 +79,9 @@ def fx_session_maker_factory(engine: AsyncEngine) -> async_sessionmaker[AsyncSes
 
 
 @pytest.fixture(name="session")
-def fx_session(sessionmaker: async_sessionmaker[AsyncSession]) -> AsyncSession:
-    return sessionmaker()
+async def fx_session(sessionmaker: async_sessionmaker[AsyncSession]) -> AsyncSession:
```

Comment:
Shouldn't the type hint be changed now that this is yielding a value and not returning?

```suggestion
async def fx_session(sessionmaker: async_sessionmaker[AsyncSession]) -> typing.AsyncGenerator[AsyncSession, None]:
```

Reply:
Good one, thanks.